### PR TITLE
add landsat-9 support

### PR DESCRIPTION
**Pull request by uchchwhash**
Hi Josh,

Could you please check if these defaults make sense to you? I have marked the ones I am unsure about with a TODO.
Most of the other numbers are from this message from Fuqin:

> Satellite orbit radius (Semi major axis):  7080640.498 (m)
> Satellite orbital inclination: 98.2269 (degree)
> Satellite angular velocity: 0.0010596442 (rad sec-1)
>
> The data are based on the following TLE information:
> 1 49260U 21088A   22184.18399685  .00000143  00000-0  41831-4 0  9995
> 2 49260  98.2269 254.2092 0000999 100.6267 259.5044 14.57115389 40571
>
> Here are also some information: https://www.n2yo.com/satellite/?s=49260

Regards and thanks,
Imam

### CODE REVIEW

File: wagl/acquisition/sensors.json [hunk 1]
```diff
@@ -521,6 +521,230 @@
...
+                    "spectral_filter_name": "landsat9_vsir.flt",
+                    "spectral_range": [2600, 349, -1],
+                    "reflectance_adjustment": 0.0365,
+                    "brdf_datasets": ["BRDF_Albedo_Parameters_Band3"],
+                    "supported_band": true
+                },
+                "3": {
+                    "alias": "Green",
+                    "band_type": "REFLECTIVE",
+                    "band_name": "BAND-3",
```
Comment (sixy6e) [inline wagl/acquisition/sensors.json:552]: I'd double-check with Mark and Fuqin on these values. They differ from [USGS' website](https://www.usgs.gov/landsat-missions/landsat-9)

Whilst I don't think they're used in any algorithmic sense (anymore), it's still ideal to have them correct.

**Comment by sixy6e** [comment]
The numbers look fine to me.  In the past, I pulled them from either the link you already provided or as a last resort Wikipedia.

These values are nominal anyway. It's what the system falls back to if there is no TLE information.
FYI, TLE is _Two Line Element_ but in future might (or already has) become _Three Line Element_ .

File: wagl/acquisition/landsat.py [hunk 2]
```diff
@@ -342,11 +342,99 @@ def radiance_data(self, window=None, out_no_data=-999, esun=None):
...
+        # TODO not sure about this, just copied from Landsat-8
+        # the downloaded TLE files are in /g/data/v10/eoancillarydata-2/sensor-specific
+        # they look pretty similar to Landsat-8 to me
+        self.tle_format = "L9%4d%sASNNOR.S00"
+
+        self.tag = "LS9"
+
+        # TODO I don't know what this means or how to get it
+        # the value is from Landsat-8
+        self.altitude = 705000.0
```
Comment (sixy6e) [inline wagl/acquisition/landsat.py:381]: These values are used as fallback values for when there is no TLE information.

In the past I've pulled a static snapshot from n2yo.com (which you've already posted), or from Wikipedia.  I wouldn't worry too much if they're too different as (whilst I haven't actually tested) I'm pretty confident the impact would be fairly insignificant.

**Review by sixy6e — COMMENTED** [review]
I'd double-check the wavelength ranges, as they differ slightly from what USGS publishes.

FYI the flt files were generally provided by Fuqin, and are a MODTRAN format.  The **BAND** labels eg **BAND-1** could be anything, but need to be consistent with what is populated within the _sensors.json_ file.

**Comment by sixy6e** [comment]
Feel from to change the sensor.json to whatever works.  It's probably getting a little unwieldy.  When we started, it only had 3 (LS5, LS7 & LS8).  But the more you add, potentially, the more cumbersome it'll become.

File: wagl/acquisition/sensors.json (hunk 1 above)
Comment (sixy6e) [reply wagl/acquisition/sensors.json:552]: the wavelength ranges i mean

**Comment by uchchwhash** [comment]
Thanks @sixy6e! Your help is deeply appreciated, as always.

**Review by sixy6e — APPROVED** [review]

**Pull request merged**